tsconfig.json

```diff
@@ -21,6 +21,9 @@
     "./test",
     "./swagger"
   ],
+  "exclude": [
+    "node_modules"
```

Comment:
Is this needed for some specific reason? Given that this is the default per https://www.typescriptlang.org/tsconfig#exclude

Reply:
- [x] done